Lib/test/test_launcher.py

```diff
@@ -287,7 +287,10 @@ def fake_venv(self):
         venv_exe = (venv / ("python_d.exe" if DEBUG_BUILD else "python.exe"))
         venv_exe.touch()
         try:
-            yield venv_exe, {"VIRTUAL_ENV": str(venv.parent)}
+            yield venv_exe, {
+                "SystemRoot": os.environ.get("SystemRoot"),
+                "VIRTUAL_ENV": str(venv.parent),
+            }
```

Comment:
Don't change this one. The context manager provides only the additional env elements, and all the tests should be going through `run_py` which overlays it correctly.